CONTRIBUTING.md

```diff
@@ -1,27 +1,90 @@
+
+|                   | Windows                                 | Mac OS                                  | Linux                                   |
+|-------------------|-----------------------------------------|-----------------------------------------|-----------------------------------------|
+| **x86_64**        | [![Download for Windows x86_64](https://img.shields.io/badge/Download-Windows%20x86__64-midnightblue?style=for-the-badge&logo=eclipseide)](https://www.eclipse.org/downloads/download.php?file=/oomph/products/eclipse-inst-jre-win64.exe) | [![Download for Mac OS x86_64](https://img.shields.io/badge/Download-Mac%20OS%20x86__64-midnightblue?style=for-the-badge&logo=eclipseide)](https://www.eclipse.org/downloads/download.php?file=/oomph/products/eclipse-inst-jre-mac64.dmg) | [![Download for Linux x86_64](https://img.shields.io/badge/Download-Linux%20x86__64-midnightblue?style=for-the-badge&logo=eclipseide)](https://www.eclipse.org/downloads/download.php?file=/oomph/products/eclipse-inst-jre-linux64.tar.gz) |
+| **AArch64**       | _Comming soon_  | [![Download for Mac OS AArch64](https://img.shields.io/badge/Download-Mac%20OS%20AArch64-midnightblue?style=for-the-badge&logo=eclipseide)](https://www.eclipse.org/downloads/download.php?file=/oomph/products/eclipse-inst-jre-mac-aarch64.dmg) | [![Download for Linux AArch64](https://img.shields.io/badge/Download-Linux%20AArch64-midnightblue?style=for-the-badge&logo=eclipseide)](https://www.eclipse.org/downloads/download.php?file=/oomph/products/eclipse-inst-jre-linux-aarch64.tar.gz) |
+
+
+Once you've downloaded the installer, proceed to the next step to install a project to work on.
+
+### Step 2: Install a Project to Contribute to
+
+Once you have the Eclipse Installer, you can set up both an IDE and a workspace containing the source-code of the project to contribute to. Each Eclipse project provides an individual _Setup_ file, which automates the installation and configuration process. See the [Project Guide](projects.md) for a list of Eclipse IDE projects. 
+
+**💡 Tip:** If you're on a Mac, check out the [additional installation instructions for Mac](InstallerOnMacosX.md).
+
+If you have not yet chosen a projec to contribute to, you can for example use the setup for **eclipse.platform.ui** [![Copy this link](https://img.shields.io/badge/Setup-orange)](https://raw.githubusercontent.com/eclipse-platform/eclipse.platform.ui/master/releng/org.eclipse.ui.releng/platformUIConfiguration.setup "Drag this setup URL in the Eclipse installer Banner"). This will allow you to get familiar with the contribution process.
+
+__Drag and drop the setup link in the banner__
+
+1. Start the Eclipse Installer.
+2. Drag and drop the setup link direclty in the Eclipse Installer Banner
+
+![DragAndDrop](images/DragAndDropSetup.jpg) 
+
+3. Switch to advanced mode (the installer will propose it as we will install an Eclipse IDE with project(s) inside).
+4. You can set up the variables but default values are OK.
```

Comment:
I don't think the defaults are ok.
At least one should be made aware that the `Root install folder` variable should be changed to the location where the user want's all Eclipse installations to reside:
![grafik](https://github.com/user-attachments/assets/dbae5a82-dbb7-46d6-9df4-ba97925222af)

And it should be mentioned that then the workspace and git repo will be co-loacted to the installation.

Furthermore I think it would be good to mention that as an alternative if one wants to choose the installation, workspace and git-clone location freely one should set the `Installation-`, `Workspace-` and `Git clone location rule`
to `Installed/Located in the specified absolute folder location` and then set the corresponding 
`Installation location`, `Workspace location` and `<Git repo name> clone location`.

Since these are mutliple options I think we should make this a sub-point and give two options.
4.1 - Set only the `Root install folder`
4.2 - Choose the absolute location rules and set `Installation location`, `Workspace location` and `<Git repo name> clone location` variables.

In both cases a screenshot would be good. One similar to the one above would be suitable for the first section, IMO.

Reply:
Implemented as discussed.